README.md

```diff
@@ -77,9 +77,16 @@ In order to run this project you need:
 -Command: install Ruby
 - [OFFICIAL](https://rubyinstaller.org/downloads/)
 
+### Run tests
 
-### Run test
--Command: Run through IRB / Vs Code
+To run the test change the file name for respective file
+
+```
+ ruby -S rspec spec/book_spec.rb
+ ruby -S rspec spec/classroom_spec.rb
+ ruby -S rspec spec/decorator_spec.rb
+ ruby -S rspec spec/person_spec.rb
+```
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
That indeed is a great idea. Obviously I will work on it